File: examples/hello-world/hello.go
package main

import "fmt"

func main() {
	fmt.Println("Hello World go !")

	a := 3
	b := 6
	c := a + b

	fmt.Printf("Sum of a(%d) + b(%d) is c(%d)\n", a, b, c)
}

// go run hello.go

File: examples/hello-world/hello.c
#include<stdio.h>
#include<curses.h>

void main() {
    system("clear");
    printf("Hello World C !\n");

    int a, b, c;
    a = 3;
    b = 6;
    c = a + b;

    printf("Sum of a(%d) + b(%d) is c(%d)", a, b, c);
    getch();
}

// cmd to run
// gcc -o hello hello.c -lncurses && ./hello